src/components/Common/Input/index.tsx

```
@@ -0,0 +1,87 @@
+
+const Input = ({
+  value,
+  type = "text",
+  placeholder = "입력해주세요",
+  label = "",
+  error = "에러가 발생하였습니다",
+  onChange,
+  onKeyDown,
+  ...props
+}: PropsType) => {
+  const [isError, setIsError] = useState<boolean>(false);
```

ESLint

[error] 'setIsError' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)